lib/galaxy/util/sanitize_html.py

```diff
@@ -112,17 +114,17 @@ def unknown_starttag(self, tag, attrs):
                 value = value.replace('>', '&gt;').replace('<', '&lt;').replace('"', '&quot;')
                 value = self.bare_ampersand.sub("&amp;", value)
                 # thanks to Kevin Marks for this breathtaking hack to deal with (valid) high-bit attribute values in UTF-8 feeds
-                if isinstance(value, unicode):
+                if isinstance(value, text_type):
                     try:
-                        value = unicode(value, self.encoding)
-                    except:
-                        value = unicode(value, 'iso-8859-1')
-                uattrs.append((unicode(key, self.encoding), value))
+                        value = unicodify(value, self.encoding)
+                    except Exception:
+                        value = unicodify(value, 'iso-8859-1')
```

Comment:
Hmm, it does work fine with feedparser as an external dependency for python2, but for python 3 I had to do these modifications.
https://github.com/kurtmckee/feedparser/compare/develop...mvdbeek:python3_fixes?expand=1
This is mostly a clash between system html module and html.py, plus sgmllib3k (a replacement for the deprecated sgmllib) is too stripped down for html sanitization.
Not sure how to proceed here.